css/index.css

```diff
@@ -143,4 +144,152 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
-}
\ No newline at end of file
+}
+
+/* my code */
+
+.container .header {
+	width: 800px;
+	height: 100px;
+    display: flex;
+    justify-content: space-between;
+	align-items: center;
+}
+
+.nav {
+	width: 630px;
+    height: 96px;
+	display: flex;
+    justify-content: space-around;
+    
+}
+
+.nav-button {
+	position: relative;
+	top: 40px;
+	text-decoration: none;
+}
+
+.nav-text {
+	color: black;
+}
+
+.logo {
+    position: relative;
```

Comment:
per our discussion, using margins, and flex box properties, are usually the best solution vs. adjusting with position: relative, since other elements in the flow don't know about its relative position